src/main/java/org/cbioportal/cdd/repository/topbraid/ClinicalAttributeMetadataRepositoryTopBraidImpl.java

```diff
@@ -35,9 +37,9 @@
 @Repository
 public class ClinicalAttributeMetadataRepositoryTopBraidImpl extends TopBraidRepository<ClinicalAttributeMetadata> implements ClinicalAttributeMetadataRepository {
 
-    private final static Logger logger = Logger.getLogger(ClinicalAttributeMetadataRepositoryTopBraidImpl.class);
+    private final static Logger logger = LoggerFactory.getLogger(ClinicalAttributeMetadataRepositoryTopBraidImpl.class);
 
-    private final static String GET_OVERRIDES_SPARQL_QUERY_STRING = "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#> " +
+    public final static String GET_OVERRIDES_SPARQL_QUERY_STRING = "PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#> " +
```

Comment:
We looked and thought the these no longer need to change to public members. (but considered it harmless)